isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -13,6 +34,200 @@ using namespace Isis;
+                            "onet="+tempDir.path()+"/apollo_out.net",
+                            "file_prefix="+tempDir.path()+"/"};
+
+  UserInterface ui(APP_XML, args);
+
+  jigsaw(ui);
+
+  // Test points.csv, images.csv, residuals.csv 
+
+  QString pointsOutput = tempDir.path() + "/bundleout_points.csv";
+  QString imagesOutput = tempDir.path() + "/bundleout_images.csv";
+  QString residualsOutput = tempDir.path() + "/residuals.csv";
+  ControlNet outputNet(tempDir.path()+"/apollo_out.net");
+
+  // Check for the correct header output format and csv file structure for the points.csv file
+  CSVReader::CSVAxis csvLine;
+  CSVReader line = CSVReader(pointsOutput,
+                               false, 0, ',', false, true);
+
+  int numColumns = line.columns();
+  int numRows = line.rows();
+
+  ASSERT_EQ(numColumns, 12);
+  ASSERT_EQ(numRows, 398);
+
+  // Validate the line information is correct
+  csvLine = line.getRow(0);
+  compareCsvLine(csvLine, "3-d,3-d,3-d,Sigma,Sigma,Sigma,Correction,Correction,Correction,Coordinate,Coordinate,Coordinate");
+
+  csvLine = line.getRow(1);
+  compareCsvLine(csvLine, "Point,Point,Accepted,Rejected,Residual,Latitude,Longitude,Radius,Latitude,Longitude,Radius,Latitude,Longitude,Radius,X,Y,Z");
+
+  csvLine = line.getRow(2);
+  compareCsvLine(csvLine, "Label,Status,Measures,Measures,RMS,(dd),(dd),(km),(m),(m),(m),(m),(m),(m),(km),(km),(km)");
+
+  // Compare all of the values from the network against the values in the CSV 
+  QList<ControlPoint*> points = outputNet.GetPoints();
+
+  EXPECT_EQ(numRows-3, points.length());
+
+  ControlPoint* outputPoint;
+  for (int i=3; i < numRows; i++) {
+    csvLine = line.getRow(i);
+    EXPECT_NO_THROW({
+      outputPoint = outputNet.GetPoint(csvLine[0]); 
+    }) << "Point in points.csv file is not present in the output network.";
+    EXPECT_EQ(outputPoint->GetPointTypeString().toUpper().toStdString(), QString(csvLine[1]).toStdString());
+    EXPECT_EQ(outputPoint->GetNumMeasures() - outputPoint->GetNumberOfRejectedMeasures(), csvLine[2].toInt());
+    EXPECT_EQ(outputPoint->GetNumberOfRejectedMeasures(), csvLine[3].toInt());
+    EXPECT_NEAR(outputPoint->GetResidualRms(), csvLine[4].toDouble(), 0.01);
+    SurfacePoint sp = outputPoint->GetAdjustedSurfacePoint();
+    SurfacePoint originalSP = outputPoint->GetAprioriSurfacePoint();
+    EXPECT_NEAR(sp.GetLatitude().planetocentric(Angle::Degrees), csvLine[5].toDouble(), 0.000001);
+    EXPECT_NEAR(sp.GetLongitude().positiveEast(Angle::Degrees), csvLine[6].toDouble(), 0.000001);
+    EXPECT_NEAR(sp.GetLocalRadius().kilometers(), csvLine[7].toDouble(), 0.000001);
+    EXPECT_NEAR(sp.GetLatSigmaDistance().meters(), csvLine[8].toDouble(), 0.000001);
+    EXPECT_NEAR(sp.GetLonSigmaDistance().meters(), csvLine[9].toDouble(), 0.000001);
+    EXPECT_NEAR(sp.GetLocalRadiusSigma().meters(), csvLine[10].toDouble(), 0.000001);
+
+    EXPECT_NEAR(sp.LatitudeToMeters(sp.GetLatitude().planetocentric(Angle::Radians)) - 
```

Comment:
@jessemapel This was my attempt to compare the corrected latitude and longitude in meters from the control network to the csv output, but the comparison I did here didn't quite work out. (See: 1000.0 as the tolerance.) Do you know what the appropriate numbers to compare would be and if they are even available at the ControlNetwork level? (The information that is written directly to the csv uses the correction from the bundle directly.)